configure

```diff
@@ -242,6 +242,29 @@ EOF
     echo "HAVE_LIBBPF_PERF_BUFFER__CONSUME:=y" >>"$CONFIG"
 }
 
+check_secure_getenv()
+{
+    cat >$TMPDIR/secure_getenv.c <<EOF
+#define _GNU_SOURCE
+#include <stdlib.h>
+#define XDP_OBJECT_ENVVAR "LIBXDP_OBJECT_PATH"
+int main(int argc, char **argv) {
+    char *path, **p;
+
+	path = secure_getenv(XDP_OBJECT_ENVVAR);
```

Comment:
Don't really need the `#define XDP_OBJECT_ENVVAR` or the variables; you can just do `secure_getenv("test");` and not assign the return value anywhere...